src/pages/AnalyticsPage/Chart/useCoverage.spec.js

```diff
@@ -2,11 +2,17 @@ import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
 import { renderHook, waitFor } from '@testing-library/react'
 import { graphql } from 'msw'
 import { setupServer } from 'msw/node'
-import { MemoryRouter, Route } from 'react-router-dom'
+import { MemoryRouter, Route , useParams } from 'react-router-dom'
+
+import { TierNames } from 'services/tier'
 
 import { useCoverage } from './useCoverage'
 
 jest.mock('services/charts')
+jest.mock('react-router-dom', () => ({
```

Comment:
This is an antipattern we used to do back in the day we're trying to replace. Effectively this is mocking useParams so that jest doesn't complain, but we can also rightfully provide it to emulate the router, take a look at this test for example, https://github.com/codecov/gazebo/blob/main/src/services/trial/useStartTrial.spec.tsx#L23. Could you get rid of this mock and instead provide the route parameters, seems like owner and provider, to the request instead? Lmk if you have a question